src/risknet/main.py

```diff
@@ -38,6 +37,7 @@ def get_job_args(arguments: Namespace) -> Tuple[Dict[str, str],
         job_args_tuples = [arg_str.split('=') for arg_str in
                            arguments.job_args]
         logger.info('job_args_tuples: %s' % job_args_tuples)
+        print(job_args_tuples)
```

Comment:
I'll go back on what I said earlier and indicate that we should have logging. We can do a separate PR for this though.